src/editor.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Represents a single editor instance.
+ *
+ * @class Editor
+ */
+
+CKEDITOR.define( [ 'ckeditor', 'mvc/model', 'utils' ], function( CKEDITOR, Model ) {
+	var Editor = Model.extend( {
+		/**
+		 * Creates a new instance of the Editor class.
+		 *
+		 * This constructor should be rarely used. When creating new editor instance use instead the
+		 * {@link CKEDITOR#create CKEDITOR.create() method}.
+		 *
+		 * @param {Object} element The DOM element that will be the source for the created editor.
+		 * @constructor
+		 */
+		constructor: function Editor( element ) {
+			/**
+			 * The original host page element upon which the editor is created. It is only supposed to be provided on
+			 * editor creation and is not subject to be modified.
+			 *
+			 * @readonly
+			 * @property {Object}
+			 */
+			this.element = element;
+		},
+
+		/**
+		 * Destroys the editor instance, releasing all resources used by it. If the editor replaced an element, the
+		 * element will be recovered.
```

Comment:
Hum... I don't think we're using it in ckeditor-dev... curious what @Reinmar has to say about it.